ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -20,6 +27,12 @@ export class ImportCommand extends IronfishCommand {
     path: Flags.string({
       description: 'the path to the file containing the account to import',
     }),
```

Comment:
Is it possible to check the USB devices automatically  instead of requiring a flag?  We could download the public address and see if it is already imported, if not then prompt whether they want to import from ledger. If it is already imported, we could skip this check. Could be fine to have a `--ledger` flag for forcing, but would be nice if the user doesn't need to know this exists.